pipeline/workflow-builder-dev.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/kiegroup/kogito-swf-builder-nightly:main-2024-04-08 AS builder
+FROM quay.io/kiegroup/kogito-swf-builder-nightly:latest AS builder
```

Comment:
according to the security recommendations, it isn't advised to use the latest.

Reply:
agree but this is the only image with the feature in it: https://github.com/apache/incubator-kie-kogito-runtimes/pull/3491 was merged on may 3rd but the most recent image with a fixed tag dates from April 28th

Reply:
And same goes for https://quay.io/repository/kiegroup/kogito-swf-builder?tab=tags&tag=latest